### PR TITLE
 Add presentationv3 rights, logo, requiredStatement (including tests)

### PR DESCRIPTION
**Pull request by jamesmisson**

### CODE REVIEW

File: __tests__/helper.test.ts [hunk 1]
```diff
@@ -3,6 +3,8 @@ import { ExternalResource } from '../src/ExternalResource';
 
 const blAvAuthNew = require('./fixtures/bl-av-auth.json');
 const blAvAuth = require('./fixtures/prev-auth.json');
+// const nwRightsLogoReqStatement = require('./fixtures/nw-rights-logo-reqStatement.json')
```
Comment (demiankatz) [inline __tests__/helper.test.ts:6]: Looks like this second manifest is never used. Should you delete this line and the associated fixture, or do you plan to add another test using this one?

File: src/Helper.ts [hunk 2]
```diff
@@ -324,6 +343,22 @@ export class Helper {
       manifestGroup.addItem(<IMetadataItem>metadataItem);
     }
 
+    const rights: string | null = this.manifest.getRights();
+    
+    if (rights) {
+      const item: any = {
+        label: "rights",
```
Comment (demiankatz) [inline src/Helper.ts:350]: Have you checked that this label gets internationalized correctly/appropriately? I'm not entirely sure how i18n works in this code, and there seem to be a few different patterns in play, so I can't tell at a glance if anything is potentially wrong.

**Review by demiankatz — CHANGES_REQUESTED** [review]
Thanks, @jamesmisson, I think these tests look great! Just one comment and one question...

File: src/Helper.ts (hunk 2 above)
Comment (jamesmisson) [reply src/Helper.ts:350]: The label gets replaced with a string from the IIIF data.content strings in iiif-metadata-component (l. 526), following the process that was already in place for 'license', so I'll make sure to check that in iiif-metadata-component/UV.

File: __tests__/helper.test.ts (hunk 1 above)
Comment (jamesmisson) [reply __tests__/helper.test.ts]: Thanks Demian I've removed that file and this line

**Pull request merged**